Jenkinsfile

```diff
@@ -69,6 +71,18 @@ for(i = 0; i < buildTypes.size(); i++) {
 builds.failFast = failFast
 parallel builds
 
+if (buildTypes.contains("linux")) {
```

Comment:
@raul-arabaolaza 
* Didn't you mean `buildTypes.contains("Linux")` with a capital `L`?
* What are you trying to achieve with this condition. AFAICT, this is always going to be `true` if using the line above. The `buildTypes` variable is statically defined on line 19 above.

Reply:
@batmat  Yeah, that is a bug, my failure when squashing commits, I am going to remove the entire condition anyway as both you and @jglick agree about it being useless. Obviously, I know that should be always true  (discarding bugs) but I wanted to increase robustness in case the variable is modified